tests/specs/ecosystem/ayonli/jsext/0_7_10.test

```diff
@@ -13,8 +13,53 @@ ayonli/jsext/0.7.10
 -- stdout --
 
 -- stderr --
-error: TS2503 [ERROR]: Cannot find namespace 'NodeJS'.
+error: TS4114 [ERROR]: This member must have an 'override' modifier because it overrides a member in the base class 'Map<K, V>'.
```

Comment:
Opened https://github.com/denoland/deno/pull/25854 to make this not appear when using `--all`